ludwig/experiment.py

```diff
@@ -496,6 +498,15 @@ def kfold_cross_validate(
                 raw_kfold_stats[category] = {}
             category_stats = \
                 kfold_cv_stats[fold_name]['fold_metric'][category]
+
+            # clear out predictions Tensors
+            # todo revisit to see if we can save predictions output
+            #    not enough structures to use output feature.postprocess_results()
+            keys = [key for key in category_stats
```

Comment:
if I understand correctly here category should be renamed metric, right?

Reply:
At this point, the dictionary `category_stats` has values like `loss`, `accuracy`, but also `predictions` and `probabilities`.

So it is a mixture of metrics and predicted output.  That is why I chose the a generic name `category_stats`.  Maybe a better name would be `fold_stats_predictions`.

Reply:
Got it. maybe let's call them `fold_outputs`.

Reply:
and `category` can be `output_name`.

Reply:
Incorporated both suggestions.